exercises/exercise-3.js

```diff
@@ -7,10 +7,18 @@
   Considerations: Values can be nested and datypes Number, String or Object.
 */
 
-function deepStringSum() {
-  // TODO: implement
+function deepStringSum(obj) {
+  let result = 0;
+  for (const elm in obj) {
+    typeof obj[elm] === "object"
+      ? (result += deepStringSum(obj[elm]))
```

Comment:
Ternary operator is a good option when you're having tiny logic but if this becomes complex would be better changing it to `if ... else if ... else`